pkg/operator/controller/status.go

```diff
@@ -53,6 +53,18 @@ func (r *reconciler) syncOperatorStatus() error {
 	if err != nil {
 		return fmt.Errorf("failed to get operator state: %v", err)
 	}
+
+	related := []configv1.ObjectReference{}
+	for _, ingress := range ingresses {
+		related = append(related, configv1.ObjectReference{
+			Group: operatorv1.GroupName,
+			Resource: "IngressController",
+			Namespace: r.Namespace,
+			Name: ingress.Name,
+		})
+	}
+	co.Status.RelatedObjects = related
```

Comment:
This should append to the `[]configv1.ObjectReference` that is defined in `initializeClusterOperator`.

Reply:
I see, good catch

Reply:
Fixed